apps/web/pages/contact.vue

```diff
@@ -1,29 +1,48 @@
 <template>
   <NuxtLayout name="default">
-    <!-- TODO: remove this code when the re-captcha/back-end call is working -->
-    <div v-if="true" class="h-[500px] flex items-center justify-center">
-      <h1>{{ $t('coming-soon') }}</h1>
-    </div>
-    <div v-else class="md:max-w-[677px] mx-auto px-4 pt-4 pb-20 md:px-0 md:mt-4">
+    <div class="md:max-w-[677px] mx-auto px-4 pt-4 pb-20 md:px-0 md:mt-4">
       <h1 class="font-bold mb-10 typography-headline-3 md:typography-headline-2">
         {{ $t('contact.contact') }}
       </h1>
       <p class="mb-10">{{ $t('contact.contactShopMessage') }}</p>
 
-      <form data-testid="contact-form" class="flex flex-col rounded-md gap-4" @submit.prevent="onSubmit">
-        <label>
-          <UiFormLabel class="mb-1">{{ $t('contact.form.nameLabel') }} *</UiFormLabel>
-          <SfInput name="name" type="text" v-model="contact.name" required />
-        </label>
-        <label>
-          <UiFormLabel class="mb-1">{{ $t('contact.form.emailLabel') }} *</UiFormLabel>
+      <form data-testid="contact-form" class="flex flex-col rounded-md gap-4" @submit.prevent="onSubmit" novalidate>
+        <div class="">
+          <label>
+            <UiFormLabel class="mb-1">{{ $t('contact.form.nameLabel') }} *</UiFormLabel>
+            <SfInput
+              v-bind="nameAttributes"
+              name="name"
+              type="text"
+              v-model="name"
+              :invalid="Boolean(errors['name'])"
+            />
+          </label>
+          <div class="h-[2rem]">
+            <VeeErrorMessage as="div" name="name" class="text-negative-700 text-left text-sm pt-[0.2rem]" />
```

Comment:
having to enforce ```pt-[0.2rem]``` is probably an indication that the layout should be refactor, please remove&adjust accordingly